styles.css

```diff
@@ -22,19 +59,68 @@ body {
     display: none;
   }
 
-  body {
-    background-color: #e5e5e5;
-    margin: 0;
-    padding: 0;
+  .desktop-popup {
+    position: fixed;
+    top: 0;
+    margin: 5%;
+    background-color: #fff;
+    z-index: 5;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    width: 90%;
+    height: 100vh;
+    border-radius: 16px;
+    padding: 2%;
+    -webkit-backdrop-filter: blur(10px);
   }
```

Comment:
- [ ] Great work making the popup scrollable. However, I noticed that the content and the popup itself are being cut off at the bottom. I would advise you to adjust it so that all contents can be seen. 💻 
- [ ] The buttons on the desktop view are not identical to that of the Figma template provided. Kindly make them identical as that is part of the requirement for this miles stone. 😎 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/218089803-a821db62-24c6-415b-a692-67438753fe69.png)

### Figma Design

![image](https://user-images.githubusercontent.com/77106962/218090095-8c7418de-475d-457a-a734-2f1c7a98cc7e.png)

### Your Design (Buttons on Desktop View)

![image](https://user-images.githubusercontent.com/77106962/218090259-95713ebb-ed40-40f1-beb4-593507d99ce3.png)

### Figm Design (Buttons on Desktop View)

![image](https://user-images.githubusercontent.com/77106962/218090364-87daa421-4c43-4c68-a361-2953aaa24a03.png)